inference-engine/src/inference_engine/include/ie/multi-device/multi_device_config.hpp

```diff
@@ -33,6 +33,6 @@ namespace MultiDeviceConfigParams {
  * @brief Device Priorities config option, with comma-separated devices listed in the desired priority
  */
 DECLARE_MULTI_CONFIG_KEY(DEVICE_PRIORITIES);
-
+DECLARE_CONFIG_KEY(NETWORK_PRIORITY);
```

Comment:
good point

Reply:
Agree to align the naming from different plugins. 

Since KEY_AUTO_... had been used by CCG MLS case, could we also use KEY_AUTO_... in our AUTO plugin? @myshevts

Reply:
its too late to align/propose the OV-level MODEL_PRIORITY (which only AUTO and GPU will support) for the 22.1. Also the GPU is already  about 'execution' (e.g. loading speed, and then inference scheduling) priorities, while the AUTO is device selection only. 
for 22.2 it would be great to have the OV level notion 'PRIORITY' to avoid device name prefix, at least for the AUTO